internal/controller/warehouses/images.go

```diff
@@ -109,17 +110,19 @@ func getImageRefs(
 	allowTagsRegex string,
 	ignoreTags []string,
 	platform string,
+	insecureSkipVerify bool,
```

Comment:
This was my mistake and not yours, but maybe worth fixing while touching this code?

We should probably just take the whole sub as an arg instead of all the fields individually.

Reply:
This had caught my 👀 as well, and I have changed it now.